### DISCUSSION TITLE
error-en-mvn-test

### DISCUSSION THREAD

**Discussion by ChavoMaiolo** [dds-jv/foro#358] [Consultas de herramientas] [answered]
Buenas creo esta discussion para consultar un error en un plugin al ejecutar el comando ` mvn test` que al ejecutar me dice lo siguiente:

![image](https://user-images.githubusercontent.com/62672266/165621587-668011c5-8284-4fb3-93c2-c593452bc907.png)

El tp ya lo termine, agregue test y los corrí con el **intellliJ** y con **maven**. Pero vi que es necesario correr el comando antes de entregar así que por las dudas consulto. Desde ya, gracias

**Comment by gastonprieto** [comment, marked as answer]
hola @ChavoMaiolo, lo que veo es que por algun motivo maven no ve los tests que creaste, en que carpeta los pusiste?

**Reply by ChavoMaiolo** [reply]
Están en el archivo que ya ustedes habían implementado "PrendaTest" para incluso evitar este tipo de problemas, pero desde un principio arrojo este error en ese plugin. Si desde la sección de maven ejecuto los test funcionan, pero es el comando `mvn test` el que invocando a ese plugin `surefire: 2.22.2` da lugar a error.

**Reply by gastonprieto** [reply]
hola, raro, mirenlo con tu ayudante en la entrega, porque por lo que veo de tu entrega tomo bien los tests, así que parece un problema de configuración en tu computadora.

**Reply by gastonprieto** [reply]
lo pudieron ver @ChavoMaiolo ?

**Reply by gastonprieto** [reply]
lo cierro por falta de respuesta
